src/ZenithAngleCalc.jl

```diff
@@ -204,103 +28,88 @@ function instantaneous_zenith_angle(date::DateTime,
     ϕ = deg2rad(latitude)
     γ = deg2rad(obliquity)
     ϖ = deg2rad(perihelion)
-    ecc = eccentricity
+    e = eccentricity
 
-    TA = true_anomaly(date)
+    # mean anomaly at vernal equinox, radians (3.11)
+    β = (1-e^2)^(0.5)
+    MAve = -ϖ + (e + e^3/4)*(1+β)*sin(ϖ) - 0.5*e^2*(0.5+β)*sin(2*ϖ) + 0.25*e^3*(1/3+β)*sin(3*ϖ)
+
+    # mean anomaly, radians (3.10)
+    julian_day_abs = datetime2julian(date)
+    julian_day_ref = 2451545.0 # Jan 1, 2000 at 12hr
```

Comment:
We should make this a parameter (called epoch), so it is easy to adjust for other planets.